en/components/grids_templates/row_drag.md

```diff
@@ -323,6 +333,17 @@ The result of the configuration can be seem below:
 <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="hierarchical-grid-row-drag-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 <div class="divider--half"></div>
+
+The following sample demonstrates row reordering in the hierarchical grid.
```

Comment:
I think this would be better as:
```suggestion
The following sample demonstrates how to configure row reordering in the hierarchical grid using row dragging.
```

Reply:
If you think the above wording words, please also apply it to the other 2 grid paragraphs (grid and tree-grid)